src/main/java/frc/robot/subsystems/LiftSubsystem.java

```diff
@@ -83,6 +91,41 @@ public Command disengageBrake() {
         });
     }
 
+    public Command autoMoveLift(Heights target) {
```

Comment:
Now that we have a working `Command`, refer to [the PIDCommand documentation](http://first.wpi.edu/FRC/roborio/release/docs/java/edu/wpi/first/wpilibj/command/PIDCommand.html) and see if you can convert this to use `PIDCommand` instead.